src/client/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -444,6 +444,11 @@ internal override void Validate()
             {
                 throw new InvalidOperationException(MsalErrorMessage.InvalidRedirectUriReceived(Config.RedirectUri));
             }
+
+            if (Config.IsBrokerEnabled && Config.IsSsoPolicyEnabled)

```

Comment:
>IsBrokerEnabled [](http://example.com/codeflow?start=23&length=15)

I don't believe we need this check.
Setting broker enable is not contradict with the sso policy. (Especially in non-windows platform)

I see this check has more harm than good.

Reply:
Removed this now after our offline discussion